libvirt/image/Makefile

```diff
@@ -68,17 +68,38 @@ STATIC_LIB_BUILDER = ../../../kata-containers/ci/install_libseccomp.sh
 STATIC_LIB_DIR     = $(abspath staticlib)
 STATIC_LIB         = $(STATIC_LIB_DIR)/kata-libseccomp/lib/libseccomp.a
 
-build: $(IMAGE_FILE)
+build: ubuntu-build
+
+rhelcheckvars:
+ifndef RHEL_IMAGE_URL
+	$(error "RHEL_IMAGE_URL is not defined")
+endif
+ifndef RHEL_IMAGE_CHECKSUM
+	$(error "RHEL_IMAGE_CHECKSUM is not defined")
+endif
+
+rhel-opts: | rhelcheckvars
+	$(eval OPTS := -var qemu_image_name=${IMAGE_FILE} \
+	-var cloud_image_url=${RHEL_IMAGE_URL} \
+	-var disk_size=11144 \
+	-var ssh_username=cloud-user \
+	-var cloud_image_checksum=${RHEL_IMAGE_CHECKSUM} rhel)
+
+rhel-build: rhel-opts $(IMAGE_FILE)
+
+ubuntu-opts:
+	$(eval OPTS := -var qemu_image_name=${IMAGE_FILE} \
+	-var cloud_image_url=${UBUNTU_IMAGE_URL} \
+	-var cloud_image_checksum=${UBUNTU_IMAGE_CHECKSUM} ubuntu)
+
+ubuntu-build: ubuntu-opts $(IMAGE_FILE)
```

Comment:
@whitebrandy I feel this change will make it harder to adopt the common Makefile/files that was introduced by @jtumber-ibm to avoid duplication.

Can we avoid introducing new build targets. How about adding a variable like PODVM_DISTRO  and based on it's value (ubuntu/rhel/centos/debian etc) set the required options for packer build. 

The packer target is modified to use the options (as you did it in this PR)